lib/stdlib/doc/src/io.xml

```diff
@@ -332,18 +331,53 @@ ok
 [{a,[97]},
  {b,[98]}]
 ok</pre>
-            <p>Binaries that look like UTF-8 encoded strings are
-              output with the string syntax if the Unicode translation
-              modifier is specified:</p>
+            <p>The Unicode translation modifier <c>t</c> specifies how to treat
+              characters outside the Latin-1 range of codepoints, in
```

Comment:
I'm not sure this punctuation is correct. Should it be "codepoints in atoms, strings, and binaries."?

Reply:
Only for those who think that the "Oxford comma" style needs to be followed slavishly even when there are no real sources of confusion. This article puts it well - "our obsession with the Oxford Comma is more of a fetish than a reasonable defense of an important part of English grammar": https://www.empirestatetribune.com/est/opinion/dean-graham/11/11/2015/its-time-to-let-go-of-the-oxford-comma

Reply:
I take it that  "correct" was not the correct word to use. The commas should be put as I wrote only because it is the style deemed correct by the guidelines for Erlang/OTP written by the technical writers Kenneth hired a few years ago. The guidelines don't seem to be available online; I think they should be.